hack/tools/release/notes.go

```diff
@@ -89,9 +93,38 @@ func firstCommit() string {
 	return string(bytes.TrimSpace(out))
 }
 
+// Since git doesn't include the last day in rev-list we want to increase 1 day to include it in the interval.
+func increaseDateByOneDay(date string) (string, error) {
+	layout := "2006-01-02"
+	datetime, err := time.Parse(layout, date)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
```suggestion
```
let's not log and return on both levels (should give us a duplicate log if I see correctly)

Reply:
Good catch!